scripts/bb_sim.py

```diff
@@ -314,4 +346,9 @@ def unfinished():
     bin_data.seek(bin_seek_vsite[i])
     vs30.tofile(bin_data)
 bin_data.close()
+logger.debug("Completed {} stations".format(len(stations_todo)))
+
 print("Process %03d of %03d finished (%.2fs)." % (rank, size, MPI.Wtime() - t0))
+comm.Barrier()
```

Comment:
what is the point of this, and the barrier added before this?

Reply:
To avoid rank_0 arrives the last line first and claim all simulation is completed while other ranks are still running

Reply:
Added comments to explain why